src/cli/serve/__tests__/fixtures/reload_logging_config/kibana.test.yml

```diff
@@ -0,0 +1,4 @@
+server:
+  port: 8274
+logging:
+  dest: src/cli/serve/__tests__/logs/first.log
```

Comment:
Wat? Not sure how the `cwd` changes this at all, but I'm removing the _log to file_ config in favor of what you mention in the next comment